pcweb/pages/docs/hosting/self_hosting.py

```diff
@@ -119,49 +133,43 @@ def self_hosting():
             rx.code("rxconfig.py"),
             ":",
         ),
-        doccode(
-            """config = rx.Config(
-    app_name="app",
-    api_url="0.0.0.0:8000",
-    bun_path="/app/.bun/bin/bun",
-    db_url="sqlite:///reflex.db",
-)
-""",
-        ),
+        doccode(docker_config),
         doctext(
             "You can find the ",
             rx.code("requirements.txt"),
             " in the ",
             rx.code("docker-example"),
             " folder of the project too.",
         ),
+        doctext("The project structure should looks like this:"),
+        doccode(project_structure, language="bash"),
         doctext(
-            "The project structure should looks like this:",
-        ),
-        doccode(
-            """hello
-├── .web
-├── assets
-├── hello
-│   ├── __init__.py
-│   └── hello.py
-├── rxconfig.py
-├── Dockerfile
-└── requirements.txt""",
-            language="bash",
+            "After all changes have been made, the container image can now be created as follows."
         ),
+        doccode(docker_build_example, language="bash"),
+        doctext("Finally, you can start your Reflex container service as follows."),
+        doccode(docker_run_example, language="bash"),
+        subheader("Optimization"),
         doctext(
-            "After all changes have been made, the container image can now be created as follows.",
+            "When self-hosting a Reflex app, you might encounter the need to improve its performance. ",
         ),
-        doccode(
-            """$ docker build -t reflex-project:latest .""",
-            language="bash",
+        doctext(
+            "Once you have a public URL leading to your app, you can use tools like ",
```

Comment:
i don't think a public url is necessary for lighthouse benchmarking... but the hosting mechanism may influence the score.

Reply:
it was necessary with the plugin, seems like the devtools integration is better.